update the test for init.R

kar-agg-gen opened the pull request:
https://3.basecamp.com/3775257/buckets/16965597/todos/3071529176
Update initDGEobj() function to make compatible with R version> 4.0

kar-agg-gen committed:
update the test for init.R

kar-agg-gen committed:
merge back from upstream

Merge remote-tracking branch 'DGEobj-upstream/tests' into tests

# Conflicts:
#	tests/testthat/test_init.R

cb4ds commented:
tests are failing in travis, please check and fix.

cb4ds commented on tests/testthat/test_init.R:
```diff
@@ -1,5 +1,44 @@
 
 
-test_that('init.R: ', {
+test_that('init.R: initDGEobj', {
```
use the function name please: initDGEobj()

cb4ds commented on tests/testthat/test_init.R:
```diff
@@ -1,5 +1,44 @@
+    rowData    <- getItem(DGEobj, "peptideAnnotation_orig")
+    colData    <- getItem(DGEobj, "design_orig")
+    level      <- "gene" # peptide level is not available
+    customAttr <- list(PID       = "20201015-0001",
+                       XpressID  = "12345",
```
Do not use or refer to PID or XpressIDs

kar-agg-gen committed:
update the title and removed the pid and xpressid

kar-agg-gen committed:
putting assignment inside expression

kar-agg-gen committed:
testing checks

kar-agg-gen committed:
reverting changes

cb4ds commented:
Note - changed the target to develop

kar-agg-gen committed:
adding show_failure function to debug the fail

kar-agg-gen committed:
Merge branch 'develop' into tests

kar-agg-gen committed:
update the test

kar-agg-gen committed:
removing message

kar-agg-gen committed:
Updating test for different Rversion

kar-agg-gen committed:
checking without message

kar-agg-gen committed:
Merge remote-tracking branch 'DGEobj-upstream/develop' into tests-checks-kar

kar-agg-gen committed:
add unit test for init.R and fixing function throwing warning in R > 4.0

kar-agg-gen committed:
Merge pull request #8 from kar-agg-gen/tests-checks-kar

updating init.R and unit-tests

kar-agg-gen committed:
merge back from tests

Merge branch 'tests' into update-tests-kar

# Conflicts:
#	tests/testthat/test_init.R

kar-agg-gen committed:
update test

kar-agg-gen committed:
Merge pull request #5 from kar-agg-gen/update-tests-kar

Fixing init.R test
